extensions/dicom-seg/.webpack/webpack.dev.js

```diff
@@ -0,0 +1,8 @@
+const path = require('path');
```

Comment:
@JamesAPetts should the extension be `dicom-seg` or `dicom-segmentation`? Does it matter that it's a cornerstone flavored `dicom-seg`?

Reply:
> dicom-seg or dicom-segmentation
Hmm, probably `dicom-segmentation` if we are being pedantic actually, `SEG` is just the modality whilst the P10 is a DICOM Segmentation IOD. However the other OHIF maintained extensions aren't exactly named in any kind of convention.

> Does it matter that it's a cornerstone flavored dicom-seg?
I don't get what you mean by this.

Reply:
@dannyrb this is pending? ill update if necessary

Reply:
@JamesAPetts, that this extension has a hard dependency on cornerstone-tools. I wasn't sure if the naming mattered, or if the intent is for it to _eventually_ work so long as its provided that it need by the platform.

@igoroctaviano, I'll let @JamesAPetts determine if a rename is worth the effort.

Reply:
Lets go with `dicom-segmentation` :)